x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/create_step.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState } from 'react';
+import { EuiButton, EuiCheckbox, EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { CreateAnalyticsFormProps } from '../../../analytics_management/hooks/use_create_analytics_form';
+import { Messages } from '../../../analytics_management/components/create_analytics_form/messages';
+import { ANALYTICS_STEPS } from '../../page';
+import { BackToListPanel } from '../back_to_list_panel';
+
+interface Props extends CreateAnalyticsFormProps {
+  step: ANALYTICS_STEPS;
+}
+
+export const CreateStep: FC<Props> = ({ actions, state, step }) => {
+  const { createAnalyticsJob, startAnalyticsJob } = actions;
+  const {
+    isAdvancedEditorValidJson,
+    isJobCreated,
+    isJobStarted,
+    isModalButtonDisabled,
+    isValid,
+    requestMessages,
+  } = state;
+
+  const [checked, setChecked] = useState<boolean>(true);
+
+  if (step !== ANALYTICS_STEPS.CREATE) return null;
+
+  const handleCreation = async () => {
+    await createAnalyticsJob();
+
+    if (checked) {
+      startAnalyticsJob();
+    }
+  };
+
+  return (
+    <Fragment>
+      <Messages messages={requestMessages} />
+      {!isJobCreated && !isJobStarted && (
+        <EuiFlexGroup gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiFormRow
+              helpText={i18n.translate(
+                'xpack.ml.dataframe.analytics.create.startCheckboxHelpText',
+                {
+                  defaultMessage:
+                    'If unselected, transform can be started later by returning to the transforms list.',
```

Comment:
Is this a copy and paste error? It is referring to transforms - shouldn't it be e.g. `job` or `analytics job`?